interpreter/subnet/src/main/scala/io/buoyant/transformer/SubnetGatewayTransformer.scala

```diff
@@ -63,7 +63,10 @@ class GatewayTransformer(
           // select the gateway addresses that share a subnet with addr
           gatewayAddrs.filter(gatewayPredicate(addr, _))
         }
-        Addr.Bound(selected, meta)
+        if (selected.isEmpty)
+          Addr.Neg
```

Comment:
I hesitate because returning Neg here doesn't necessarily mean something is wrong.  For example, you could imagine building a fallback dtab that attempts to route to a daemonset but falls back to something else if no matching daemonset pods exist.  In that case we'd log on every request even though the requests get successfully routed to the fallback.